libs/net/dal/Models/ContentFileReference.cs

```diff
@@ -59,7 +59,7 @@ public class ContentFileReference : IReadonlyFileReference
     /// <summary>
     /// get - The file to upload with the file reference.
     /// </summary>
-    public IFormFile File { get; }
+    public IFormFile? File { get; }
```

Comment:
This issue was introduced in the Video/Audio editor PR.  I'm not certain the best way to resolve it.  For now I resolved the linting issue.